test/SparqlParser-test.js

```diff
@@ -1,15 +1,18 @@
 var SparqlParser = require('../sparql').Parser;
 
-var fs = require('fs'),
-    expect = require('chai').expect;
-
-var factory = require('./TestDataFactory');
+var fs = require('fs');
+    // expect = require('chai').expect;
+var expect = require("expect");
+var toEqualParsedQuery = require("../test/matchers/toEqualParsedQuery");
+expect.extend({
+  toEqualParsedQuery,
+});
 
 var queriesPath = __dirname + '/../queries/';
 var parsedQueriesPath = __dirname + '/../test/parsedQueries/';
 
 describe('A SPARQL parser', function () {
-  var parser = new SparqlParser(null, null, factory);
+  var parser = new SparqlParser(null, null, null);
```

Comment:
I assume this is a mistake? Should the factory not be passed explicitly? (Also later on)

Reply:
By passing null, I let the parser use it's default factory. That's the factory I would want to pass as the argument anyway, so passing null is just to save us an import.

Reply:
Ok, in that case this line can be simplified to `new SparqlParser()`.

Reply:
@RubenVerborgh I think there was a reason why we wanted to pass `TestDataFactory` here?